packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_control_pane.dart

```diff
@@ -132,3 +159,27 @@ class _DiffDropdown extends StatelessWidget {
     );
   }
 }
+
+class _SnapshotSizeView extends StatelessWidget {
+  const _SnapshotSizeView({Key? key, required this.footprint})
+      : super(key: key);
+
+  final MemoryFootprint footprint;
+
+  @override
+  Widget build(BuildContext context) {
+    final items = <String, int>{
+      'Dart Heap': footprint.dart,
+      'Reachable': footprint.reachable,
+    };
+    return Text(
+      items.entries
+          .map<String>(
+            (e) => '${e.key}: ${prettyPrintBytes(e.value, includeUnit: true)}',
+          )
+          .join(' | '),
```

Comment:
This works, but consider using a vertical divider instead of text.

Reply:
added TODO